extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/common/KubernetesPeonClient.java

```diff
@@ -87,6 +105,8 @@ public JobResponse waitForPeonJobCompletion(K8sTaskId taskId, long howLong, Time
                           howLong,
                           unit
                       );
+      long duration = System.currentTimeMillis() - start;
+      emitK8sPodMetrics(job, "peon/running/time", duration);
```

Comment:
How is this different than the `task/run/time` metric?

Reply:
i think you would probably want to include the startup time as well, maybe it makes sense to emit this in KubernetesPeonLifecycle instead?

Reply:
Removed, `task/run/time` metric will include job startup and running time.